tests/integration/inhibit-if-kmods-is-not-supported/ansible/kmods.yml

```diff
@@ -0,0 +1,5 @@
+- hosts: all
+#  tasks:
+#    - name: Install convert2rhel deps
+#      pip:
+#        name: ["pytest", "pytest-cov"]
```

Comment:
Since the content is commented out (the installation of the deps should happen in tests/ansible_collections/basic_setup.yml, right?), you can remove the whole file.